tests/test-amp-blacklist-sanitizer.php

```diff
@@ -102,6 +107,26 @@ public function get_data() {
 				'<a href="http://example.com" target="boom">Link</a>',
 				'<a href="http://example.com">Link</a>',
 			),
+
+			'a_with_href_mailto' => array(
+				'<a href="mailto:email@domain.com">Link</a>',
```

Comment:
`mailto` should be okay (it's included in an example here: https://github.com/ampproject/amphtml/blob/eb641b052d44f78b8b83fc91bc64394b7263ee5e/examples/article.amp.html#L203)